syntax = "proto3";

package second.protos;

import "rslo/protos/input_reader.proto";
import "rslo/protos/model.proto";
import "rslo/protos/train.proto";
// Convenience message for configuring a training and eval pipeline. Allows all
// of the pipeline parameters to be configured from one file.
message TrainEvalPipelineConfig {
  DetectionModel model = 1;
  InputReader train_input_reader = 2;
  TrainConfig train_config = 3;
  InputReader eval_input_reader = 4;
  InputReader eval_train_input_reader = 5;
}

